lib/Room.php

```diff
@@ -163,6 +163,13 @@ public function hasPassword() {
 		return $this->password !== '';
 	}
 
+	/**
+	 * @return string
+	 */
+	public function getPassword() {
```

Comment:
Please call this method from a test to avoid future breakage, e.g. after creating the `Room` here:
https://github.com/nextcloud/spreed/blob/c1d4482d507e44929037b3495cccd63fa5ba0d38/tests/php/PasswordVerificationTest.php#L47
(`getPassword()` should return `passy`).

Reply:
Added a test and using verify from the hasher to test the password as it returns the hashed value.